packages/compass-import-export/src/utils/collection-stream.js

```diff
@@ -69,57 +71,85 @@ class WritableCollectionStream extends Writable {
     this._executeBatch(callback);
   }
 
-  _executeBatch(callback) {
+  async _executeBatch(callback) {
     const documents = this.batch;
 
     this.batch = [];
 
-    this._collection().bulkWrite(
-      documents.map((doc) => ({ insertOne: doc })),
-      {
-        ordered: this.stopOnErrors,
-        retryWrites: false,
-        checkKeys: false,
-      },
-      (err, res) => {
+    const bulkWrite = promisify(
+      this._collection().bulkWrite.bind(this._collection())
```

Comment:
No need to promisify, right? `collection.bulkWrite()` returns a Promise if it doesn’t receive a callback argument